tests/framework/microvm.py

```diff
@@ -82,6 +82,10 @@ def __init__(
         )
         self.jailer_clone_pid = None
 
+        # Copy the /etc/localtime file in the jailer root
+        self.jailer.copy_into_root(
```

Comment:
If we install `tzdata` in the container we may not need to add an extra function, thus simplifying the PR.

Reply:
Installing `tzdata` would just be another way of mounting `/etc/localtime`. We still need to copy this file into the jailer, so we can't get rid of this function